app/api/modules.py

```diff
@@ -29,9 +28,6 @@ def before_patch(self, args, kwargs, data=None):
         :param data:
         :return:
         """
-        if data.get('donation_include'):
```

Comment:
Why remove this

Reply:
This Check should be present in the events api where if the donations are disabled and the organiser tries to pass something for donations, an exception should be thrown(which has already been incorporated in the code). As quoted by @CosmicCoder96, I think this check shouldn't be present here.

Reply:
yeah @iamareebjamal I believe this exact check should be placed in the events api, and this very exception should be raised *after* checking from the modules api if donation is enabled in the system. If the admin has not enabled it, then this field should not be present in the request body at all.